ibc-clients/cw-context/src/context/mod.rs

```diff
@@ -219,58 +218,6 @@ where
         client_update_height_path.leaf().into_bytes()
     }
 
-    /// Returns the genesis metadata by iterating over the stored consensus
-    /// state metadata.
-    pub fn get_metadata(&self) -> Result<Option<Vec<GenesisMetadata>>, ContractError> {
```

Comment:
I've removed metadata query. Read [my comment](https://github.com/cosmos/ibc-rs/issues/1277#issuecomment-2229161396) on this.

Reply:
Thanks for the context. Makes sense.